rcon/hooks.py

```diff
@@ -232,6 +241,9 @@ def handle_on_connect(rcon, struct_log):
             )
 
     timestamp = int(struct_log["timestamp_ms"]) / 1000
+
+    kick_if_low_playtime(rcon=rcon, struct_log=struct_log, steam_id_64=steam_id_64)
```

Comment:
I'd suggest just adding the `@on_connected` on the `kick_if_low_playtime` function directly
It provided some isolation in case there a problem and won't break the rest of the features in that hook.